app/controllers/sprangular/accounts_controller.rb

```diff
@@ -47,8 +47,11 @@ def spree_user_params
   end
 
   def render_user
+    serializer = params[:serializer] == 'full' ? Sprangular::UserSerializer : Sprangular::LiteUserSerializer
```

Comment:
Line is too long. [108/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.